flow_reorganize_symlinks/scaffold_symlink.cwl

```diff
@@ -13,9 +13,10 @@ arguments:
     position: 1
     valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_genomic_fasta[0])/$(inputs.genome_fasta_name[0]).gz
     #readme file
+    valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_genomic_fasta[0])/readme.txt.gz
```

Comment:
cwl-runner didn't like this line when I ran it. It worked fine when I commented it out, though.

Reply:
@HsiuKangHuang Here's the error message that I got:
```
(venv)[mpoelchau@gmod-dev /app/data/monica_data/Organism_Onboarding]$ cwl-runner CreateSymlink-workflow.cwl test-symlink.yml
/app/data/cwltool/venv/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
Please upgrade your Python as the Python 2.7 version of cwltool won't be
maintained after that date.

  """,  category=CWLToolDeprecationWarning)
INFO /app/data/cwltool/venv/bin/cwl-runner 1.0.20191225192155
INFO Resolved 'CreateSymlink-workflow.cwl' to 'file:///app/data/monica_data/Organism_Onboarding/CreateSymlink-workflow.cwl'
ERROR I'm sorry, I couldn't load this CWL file, try again with --debug for more information.
The error was: while constructing a mapping
  in "file:///app/data/monica_data/Organism_Onboarding/flow_reorganize_symlinks/scaffold_symlink.cwl", line 11, column 5
found duplicate key "valueFrom" with value "$(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_genomic_fasta[0])/readme.txt.gz" (original value: "$(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_genomic_fasta[0])/$(inputs.genome_fasta_name[0]).gz")
  in "file:///app/data/monica_data/Organism_Onboarding/flow_reorganize_symlinks/scaffold_symlink.cwl", line 16, column 5

To suppress this check see:
    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys

Duplicate keys will become an error in future releases, and are errors
by default when using the new API.
```

I verified that the files in question (readme.txt.gz and $(inputs.genome_fasta_name[0]).gz) are in the directory.